packages/opentelemetry-tracing/src/config.ts

```diff
@@ -48,7 +47,7 @@ export const DEFAULT_CONFIG = {
  * @param env optional, by default uses getEnv(), but allows passing a value to reuse parsed environment
  */
 export function buildSamplerFromEnv(
-  env: Required<ENVIRONMENT> = getEnv()
+  env: ReturnType<typeof getEnv> = getEnv()
```

Comment:
Wouldn't it be better to just expose the `Environment` type?

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/2178#issuecomment-831312205

Reply:
> > We could also export ENVIRONMENT in core. Effectivly it's exported already.
> 
> I checked and i couldn't find it, my version has the benefit of not needing the update the type if the getEnv return something else

It's in the file that was previously referenced. Using the actual type instead of `ReturnType` may also produce better compilation failure messages, especially for customers not using the latest typescripts.

Reply:
Fixed https://github.com/open-telemetry/opentelemetry-js/pull/2178/commits/defdbf1306e07699af995e4af38c0d740a3cdb09